spec/controllers/password_resets_controller_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+RSpec.describe PasswordResetsController, :type => :controller do 
+
+  describe "GET show" do
+    it "renders show template if the token is valid" do
+      alice = Fabricate(:user)
+      alice.update_columns(token: "12345")
+      get :show, id: '12345'
+      expect(response).to render_template :show
+    end
+
+    it "sets @token" do
+      alice = Fabricate(:user)
+      alice.update_columns(token: "12345")
```

Comment:
I noticed you do that for specs below, so this comment works for below specs so i don't have to copy it there.